esphome/components/remote_base/rc5_protocol.h

```diff
@@ -9,6 +9,7 @@ namespace remote_base {
 struct RC5Data {
   uint8_t address;
   uint8_t command;
+  uint8_t toggle;
```

Comment:
Apologies for the delay, should this just be a `bool` instead of `uint8_t`?
```suggestion
  bool toggle;
```